### ISSUE TITLE
liveTracker.seekToLiveEdge() no longer un-pauses

### ISSUE THREAD

**Issue by Wayne-Morgan** [videojs/video.js#7619]
## Description
I am updating to 7.18.0 after a long time on a much older version
I notice that when seeking to the live edge from paused, playback no longer re-starts.
This functionality looks like it was changed in #6405, but there was no explanation as to why.
The documentation in [live.md](https://github.com/videojs/video.js/blob/main/docs/guides/live.md#seektoliveedge) gives the impression that this should still happen.

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.
1. Play a live stream
2. Pause
3. Click the "live" button to seek to the live edge

### Expected
Playback continues from the live edge

### Actual
Seeks to live edge but remains paused

#### videojs
what version of videojs does this occur with?
I've seen the issue with 7.18.0, but it's probably everything since 7.8.0
#### browsers
what browser are affected? all?
#### OSes
what platforms (operating systems and devices) are affected? all?
### plugins
are any videojs plugins being used on the page? If so, please list them below.
videojs-shaka

**Comment by misteroneill** [comment]
You're right, it does look like it was removed in that PR. I'm not sure if it was intentional or not, but we have to assume it was. @brandonocasey do you happen to remember anything about this?

**Comment by brandonocasey** [comment]
I think it was likely done to preserve the state of the player. IE if you are paused and seek to live you stay paused, if you play and seek to live you play. Perhaps we should just add a liveTracker option though and default to true to bring this functionality back.

**Comment by david-hm-morgan** [comment]
Commenting to keep this open: https://videojs.com/guides/live/#seektoliveedge does contradict this behaviour.
